### PR TITLE
NODE-1742: Transactions generator

### PR DESCRIPTION
**Pull request by chepiov**
JIRA: https://jira.wavesplatform.com/browse/NODE-1742

### CODE REVIEW

**Review by phearnot — APPROVED** [review]

File: node-generator/src/main/scala/com/wavesplatform/generator/NarrowTransactionGenerator.scala [hunk 1]
```diff
@@ -53,8 +54,8 @@ class NarrowTransactionGenerator(
     val generated = (0 until (n * 1.2).toInt).foldLeft(
       (
         Seq.empty[Transaction],
-        preconditions.tradeAsset.fold(Seq.empty[IssueTransactionV2])(Seq(_)),
-        preconditions.tradeAsset.fold(Seq.empty[IssueTransactionV2])(Seq(_)),
+        preconditions.tradeAsset.fold(Seq.empty[IssueTransaction])(Seq(_)),
+        preconditions.tradeAsset.fold(Seq.empty[IssueTransaction])(Seq(_)),
         Universe.Leases,
         Seq.empty[CreateAliasTransaction]
```
Comment (AOHhiroko) [inline node-generator/src/main/scala/com/wavesplatform/generator/NarrowTransactionGenerator.scala:60]: Sorry, I do not understand these pages. Do you have any assistance or guide to understand the programming words,,such as timestamp, val, cats show, final case class, etc. (especially 51 is hard to understand)

File: node-generator/src/main/scala/com/wavesplatform/generator/Preconditions.scala
Comment (AOHhiroko) [inline node-generator/src/main/scala/com/wavesplatform/generator/Preconditions.scala:1]: HI, I read your suggestion. But I don't know how to reissue assets. could you help me with increasing the probability? 
_'ll visit my accounts in waves_

Comment (AOHhiroko) [reply node-generator/src/main/scala/com/wavesplatform/generator/Preconditions.scala:1]: I read these documents but still I  do not understand.. Could you tell me what is going on.?  
My wave's accounts are not settled down, Could I  ask  Mr. Chepiov who helped me first  in  2019 to help me now?
My google account might be easier to open. Yestery I made a mistake  to open one. thank you.
Hiroko Aoh

Comment (AOHhiroko) [reply node-generator/src/main/scala/com/wavesplatform/generator/Preconditions.scala:1]: Does this mean I am going to receive large waves token?
If so is it possible to pay reward to the waves team ?  Still I do not know what to say.

.

Comment (AOHhiroko) [reply node-generator/src/main/scala/com/wavesplatform/generator/Preconditions.scala:1]: If the way is frazil, or difficult, please do not  reissue. I do not know  but please take  appropriate amount for your work.
for waves support team, and also Git hub's team.

File: node-generator/src/main/scala/com/wavesplatform/generator/NarrowTransactionGenerator.scala (hunk 1 above)
Comment (AOHhiroko) [reply node-generator/src/main/scala/com/wavesplatform/generator/NarrowTransactionGenerator.scala:60]: > Sorry, I do not understand these pages. Do you have any assistance or guide to understand the programming words,,such as timestamp, val, cats show, final case class, etc. (especially 51 is hard to understand)

Hi, I still do not understand as a whole, what should I do.   What does boolean mean? I do not want a difficulty  comes up, So, if  ”reissue" draws  difficulty, and you think it is not neccesary, please take it back.

Comment (AOHhiroko) [reply node-generator/src/main/scala/com/wavesplatform/generator/NarrowTransactionGenerator.scala:60]: I do not understand "narrow transaction".  Is there anything which I should do?
I appreciate your help, very much. Today I opened my"waves accounts"and I restored 
one,"gracetowerbiwa." But others were not revived yet. I appreciate your help, very much.

**Review by AOHhiroko — COMMENTED** [review]
Hi, I still do not understand what ai should do..I do  not understand how to commit。
I got my security number for ViISA card yesterday and sent it to you. If possible, 
CHEPIOV, would you help me to accomplish this job? It is too much for me.
I hope  Copilot will help me, too..

**Pull request merged**